src/System.Net.Security/ref/project.json

```diff
@@ -9,9 +9,9 @@
     "System.Threading.Tasks": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.6": {
```

Comment:
See #11321.

Looks like this should be netstandard1.7 instead of netstandard1.6.  (Currently docs are wrong).